modules/nw-egress-ips-about.adoc

```diff
@@ -26,16 +26,28 @@ An egress IP address is implemented as an additional IP address on the primary n
 [id="nw-egress-ips-platform-support_{context}"]
 == Platform support
 
-While you can use the egress IP address functionality on any platform that {product-title} supports, the following platform specific considerations apply:
+Support for the egress IP address functionality on various platforms is summarized in the following table:
 
-Cloud platforms::
-Additional IP addresses must be assigned manually to appropriate nodes by an administrator.
+[IMPORTANT]
+====
+The egress IP address implementation is not compatible with Amazon AWS, Azure Cloud, or any other public cloud platform that does not support manipulation of layer 2 network traffic.
```

Comment:
I would rephrase to `or any other public cloud platform incompatible with the automatic layer 2 network manipulation required by OpenShift's egress IP feature.`. The reason is that in the future this will work, it's just that we currently have not done the necessary development for that to happen.